soft4pes/utils/conversions.py

```diff
@@ -30,24 +30,72 @@ def abc_2_alpha_beta(abc):
     return ab.T
 
 
+def alpha_beta_2_dq(alpha_beta, theta):
```

Comment:
Is the abc_2_alpha_beta transformation (lines 8 - 30) going to change? It assume n quantities instead of 1 as in the transformation below, and the dot product would like to receive a transposed 1D array (which is actually not transposed).

Reply:
Yes, that would be a nice feature